jbpm-wb-process-instance-migration/pim-service/src/main/resources/modules/com/oracle/main/module.xml

```diff
@@ -0,0 +1,9 @@
+<module xmlns="urn:jboss:module:1.1" name="com.oracle">
```

Comment:
We probably shouldn't include mysql and oracle drivers by default, including jboss modules representing them, wdyt? It should be up to a user what they decide to use and put on their classpath. I believe they should create their own module inside the jar file - which means they will have to modify the jar file. I think running just `java -cp` with the app jar and JDBC jar as a classpath won't work since Thorntail uses modules internally.

Here is what I found:
https://docs.thorntail.io/2.4.0.Final/#configuring-a-datasource_manually_thorntail